core/templates/pages/skill-editor-page/skill-editor-page.component.html

```diff
@@ -104,7 +105,7 @@
   }
 
   .skill-editor-main .navbar-tab span {
-    color: #FFF;
+    color: #333;
```

Comment:
Is the color attribute actually required here? I would imagine that the color will be taken from `.oppia-editor-warnings-error-color`.

Reply:
I also thought about that, but `<span>` color is overriding it and that's why it's not working. @kevintab95 
Thanks

Reply:
Can you explain "but `<span>` color is overriding it" please? Which CSS is conflicting with the color property in `.oppia-editor-warnings-error-color`?

Reply:
Sure,
The main problem we were having was the color of `[$ctrl.getWarningsCount()]` and the color of the `Warnings` word. So I try to post the `.oppia-editor-warnings-error-color` class there and see if it works
```
<span class="oppia-editor-warnings-count oppia-editor-warnings-error-color">
        <[$ctrl.getWarningsCount()]>
</span>
```
It didn't work as in our local CSS classes which are present within the skill file we have a class for `<span>` in which the color is `#FFF` and that's why it's dominating.
So I tried to change the color as mainly it is used mainly for the navbar purpose to `#333` and it worked.
 
Please do let me know if I am somewhere getting it wrong.
Thanks

Reply:
@kevintab95 PTAL